docs/Generators/Gemmini.rst

```diff
@@ -0,0 +1,52 @@
+Gemmini
+====================================
+
+The Gemmini project is developing a systolic-array based matrix multiplication acceleration generation for the investigation of SoC integration of such accelerators. It is inspired by recent trends in machine learning accelerators for edge and mobile SoCs.
```

Comment:
This first sentence sounds a bit off. I would reword.

Reply:
reworded